core/src/main/java/com/exadel/aem/toolkit/core/lists/utils/ListPageUtils.java

```diff
@@ -0,0 +1,50 @@
+package com.exadel.aem.toolkit.core.lists.utils;
```

Comment:
And here Apache License header missing too. Could you please add it to this file?